content/S2016/S16-event-csters-destress.md

```diff
@@ -0,0 +1,26 @@
+title: Big CSters: WiCS & FemPhys De-stressing Session
+Date: 2016-6-1
+Category: Events
+Tags: social, big csters
+Slug: big-csters-destressing-s16
+Author: Clare Ng
+Summary: Join us for a relaxing night of socializing and de-stressing.
+
+WiCS is collaborating with FemPhys to bring you a de-stressing session! We 
```

Comment:
Do we want to link to femphys' facebook page?
https://www.facebook.com/groups/705741329511034/?fref=nf